README.md

```diff
@@ -1,2 +1,37 @@
-# pdf-redaction-annotation-csharp
-This repository contains various examples to use the PDF redaction annotation
+# PDFRedactionAnnotation in C#
+The [Syncfusion .NET PDF library](https://www.syncfusion.com/document-processing/pdf-framework/net) allows removing the sensitive text and images from the PDF document using its PDF Redaction feature. This repository contains various examples to use the PDF redaction annotation.
+
+Sample name | Description
+--- | ---
+[ContentRemoval](https://github.com/SyncfusionExamples/pdf-redaction-annotation-csharp/blob/main/.NET%20Framework/ContentRemoval) | Remove the marked content from PDF document.
```

Comment:
Add the proper sample name for all the samples

Reply:
changed the sample name and description